### PR TITLE
Repair-Player.ArtificialHealth

### LINKED ISSUE #931: [BUG] Player.ArtificialHealth broken
**Describe the bug**
Setting a player's artificial health is not going to have any effect.

**To Reproduce**
``Player.ArtificalHealth = 30f;``

**Expected behavior**
That player should gain 30 ahp

**EXILED Version ("latest" is not a version):**
4.1.2

### PR DESCRIPTION
**Pull request by louis1706**
Fixes #931

### CODE REVIEW

**Comment by joker-119** [comment]
Can't .First() cause a nullref if they don't have any active processes though?

**Comment by joker-119** [comment]
Also change the target branch to dev :anger:

**Comment by louis1706** [comment]
> Can't .First() cause a nullref if they don't have any active processes though?

**Review by joker-119 — APPROVED** [review]

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -697,8 +697,14 @@ public float ArtificialHealth
         /// </summary>
         public float MaxArtificialHealth
         {
-            get => ((AhpStat)ReferenceHub.playerStats.StatModules[1])._maxSoFar;
-            set => ((AhpStat)ReferenceHub.playerStats.StatModules[1])._maxSoFar = value;
+            get => ActiveArtificialHealthProcesses?.First().Limit ?? 0;
+            set
+            {
+                if (ActiveArtificialHealthProcesses.Count() < 1)
```
Comment (sanyae2439) [inline Exiled.API/Features/Player.cs:703]: Use Any().

**Review by sanyae2439 — CHANGES_REQUESTED** [review]

**Pull request merged**